### PR TITLE
Allow proxy_read_timeout to be configured

### PR DESCRIPTION
**Pull request by thebignet**
Add possibility to configure the [proxy read timeout](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_read_timeout).

### CODE REVIEW

File: CONFIGURATION.md [hunk 1]
```diff
@@ -149,6 +150,15 @@ For more example and resulting nginx configurations take a look at the
    - http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass
 
 
+### proxyReadTimeout
+
+When set, reading a response from the proxied server will timeout at this.
+
+- **Type**: `string`
+- **Default**: 60
```
Comment (bripkens) [inline CONFIGURATION.md]:
```suggestion
- **Default**: 60s
```

File: test/expectedConfigs/timeout.conf [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  include                    mime.types;
+  default_type               application/octet-stream;
+
+  sendfile                   on;
+  keepalive_timeout          65;
+  client_max_body_size       0;
+  client_body_temp_path      /tmp/proxrox/clientBody;
+  proxy_temp_path            /tmp/proxrox/proxy;
+
+  proxy_read_timeout         120;
```
Comment (bripkens) [inline test/expectedConfigs/timeout.conf]:
```suggestion
  proxy_read_timeout         120s;
```

File: test/config_generator_test.js [hunk 3]
```diff
@@ -232,7 +232,20 @@ describe('config_generator', function() {
...
+    it('should support read timeout', function () {
+      testConfig(
+        {
+          standardServer: true,
+          tmpDir: '/tmp/proxrox',
+          serverName: 'example',
+          port: 8080,
+          root: '/var/www',
+          logDir: '/tmp/nginx-logs/',
+          proxyReadTimeout: 120,
```
Comment (bripkens) [inline test/config_generator_test.js]:
```suggestion
          proxyReadTimeout: 120s,
```

**Review by bripkens — COMMENTED** [review]
Nice change, thanks! :)

I added some suggestions for time units. Could you also please add [a changelog entry](https://github.com/bripkens/proxrox/blob/master/CHANGELOG.md#1160) under an `Unreleased` heading? Thank you!

**Comment by thebignet** [comment]
@bripkens I've pushed the requested changes. Tell me what you think about it.

**Comment by bripkens** [comment]
LGTM, thank you @thebignet! :)

**Pull request merged**